docs/src/CommutativeAlgebra/rings.md

```diff
@@ -25,15 +25,13 @@ of the coefficient ring of the polynomial ring.
 The basic constructor below allows one to build multivariate polynomial rings:
 
 ```@julia
-polynomial_ring(C::Ring, V::Vector{String}; ordering=:lex, cached::Bool = true)
+polynomial_ring(C::Ring, V::Vector{String}; cached::Bool = true)
```

Comment:
@wdecker I removed mentioning this keyword here and below in this file. I think this way, it is less confusing for the non-expert user who does not need to know about such implementation details (it is still explained in the documentation on rings which we copy from AbstractAlgebra). Is this fine?